app/Http/Controllers/UserImageController.php

```diff
@@ -55,19 +51,33 @@ public function store(Request $request, ImageService $imageService): RedirectRes
 
     }
 
-    public function edit()
+
+    public function destroy(ImageService $imageService): RedirectResponse
     {
+        try{
 
-    }
+            $userImage = Auth::user()->images->first();
 
-    public function update()
-    {
+            if (!$userImage) {
+                throw new \Exception("There is no image to delete.");
+            }
 
-    }
+            $deleteImageFromDir = $imageService->deleteImageFromDirectory($userImage->filename);
 
+            if ($deleteImageFromDir === false){
```

Comment:
if false it should mean the file was not successfully deleted. otherwise if the file does not exist, you can go ahead and delete the image record